rosbag2_transport/test/rosbag2_transport/test_play_until.cpp

```diff
@@ -154,6 +154,48 @@ TEST_F(RosBag2PlayUntilTestFixture, play_until_less_than_the_total_duration)
   EXPECT_EQ(replayed_topic1[1]->int32_value, 1);
 }
 
+TEST_F(RosBag2PlayUntilTestFixture, play_until_less_than_the_total_duration_message_order)
+{
+  auto primitive_message1 = get_messages_basic_types()[0];
+  auto primitive_message2 = get_messages_basic_types()[0];
+  primitive_message1->int32_value = 1;
+  primitive_message2->int32_value = 2;
+
+  auto topic_types = std::vector<rosbag2_storage::TopicMetadata>{
+    {1u, kTopic1Name_, "test_msgs/BasicTypes", "", {}, ""}};
```

Comment:
```suggestion
  auto topic_types = std::vector<rosbag2_storage::TopicMetadata>{
    {1u, kTopic1Name_, "test_msgs/msg/BasicTypes", "", {}, ""}};
```

Reply:
changed in 9f23136562d51e20ff1ad4f5b62df4e43c7e9f16